src/cBiff12Writer.cls

```diff
@@ -130,6 +136,21 @@ Public Function Init( _
     Init = True
 End Function
 
+Public Function AddSheet(Optional ByVal SheetName As String, Optional ByVal lColCount As Long) As Boolean
+    pvEndSheet m_oSheetPart
+    Set m_oSheetPart = m_oContainer.SheetPart(m_oContainer.SheetPartCount + 1)
+    m_oSheetPart.SheetName = IIf(Not IsMissing(SheetName), SheetName, "Sheet" & Format(m_oContainer.SheetPartCount, "0"))
```

Comment:
Using `IsMissing` for `String` optional param is not possible (works only w/ `Variant` params).

A `IIf(LenB(SheetName) = 0, ...` can be used for the intended optional string not supplied check.

Reply:
I need  my IDE  ...
I check thus tomorrow at work and then we can merge